cli/devlog.py

```
@@ -0,0 +1,125 @@
+@click.option("--story", "story_id", required=True)
+@click.option("--action", type=click.Choice(["start", "stop"]), required=True)
+@click.option("--date", type=click.DateTime(formats=["%Y-%m-%d"]), required=False)
+def record(story_id: str, action: str, date: datetime | None):
+    """Start/stop OBS recording and persist story state."""
+    date = _today(date)
+    
+    # Initialize OBSController with error handling
+    try:
+        from services.obs_controller import OBSController
+        obs = OBSController()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)